src/resizing.jl

```diff
@@ -317,6 +333,12 @@ function imresize!(resized::AbstractArray{T,N}, original::AbstractArray{S,N}) wh
     imresize!(resized, itp)
 end
 
+function imresize!(resized::AbstractArray{T,N}, original::AbstractArray{S,N}, aitp::Interpolations.Lanczos4OpenCV) where {T,S,N}
+    # FIXME: avoid allocation for interpolation
+    itp = interpolate(original, aitp)
```

Comment:
@mkitti do we have a lazy version of `interpolate`? Sorry that I don't know much about Interpolations' internal.

Reply:
`interpolate` is pretty lazy, but the details depend on a specific interpolation kernel.